ddtrace/contrib/logging/__init__.py

```diff
@@ -0,0 +1,33 @@
+"""
+The patching of standard library logging to inject tracing information
+
+Example::
+    import logging
+    from ddtrace import tracer
+    from ddtrace import patch_all; patch_all(logging=True)
```

Comment:
I think one of the approach we should use, is telling users to go through `ddtrace-run` instead of "manual patching". In that case, the `DD_LOGS_INTEGRATION=true` should land here. I would say to put the `ddtrace-run` as a first example, and leave this option as a second example.